career/working-and-collaborating-remotely.md

```diff
@@ -0,0 +1,172 @@
+
+* **Maintain and Build Your Skills** - Learning is a lifetime endeavor, especially so in development. Technologies are constantly changing, and you will need to be proactive to remain competent on your team and competitive in the job market.
+  - Pay attention to the tools, languages, and platforms your company is using. If you are unfamiliar with any of them, research them for a basic understanding of how they work
+  - When you have one-on-ones with your supervisor, treat them as professional development opportunities. Ask questions and request feedback on your performance.  Ask your supervisor about new skills you should be learning.
+  - If there is a skill you want to learn, talk to your supervisor about ways to incorporate it into the work you are already doing
+  - Most companies have a professional development policy which allows staff a certain number of hours a year to dedicate to learning new skills.  Many companies also have a budget to cover the costs
+  - Be proactive about learning new things - if there isn’t a training opportunity within your company to learn, you can often find online courses to teach you, or consider working on open source projects
+
+### Common Mistakes & Misconceptions
+- **I can work exactly where I feel like, when I feel like!** - While this may be true for experienced independent contractors, many companies will set your hours according to time zone, or at minimum require a set of hours that they know you will be working and available for meetings, calls, and reporting. Also consider that it would be unprofessional to do video conferencing from a cafe or outdoors where background noise will be an issue.
+- **Remote work sounds like a great way to be judged solely by my programming skills** - Whether this sounds good because you are an introvert or you don't like office politics, in order to grow professionally, you may have to try even harder to build your network and gain a solid professional reputation.
+- **They are giving me a laptop, now I don't need to buy a new one** - If you are thinking that now you don’t have to buy a new computer to contribute to your non-profit or build your Etsy shop, think again. Your work laptop is your company’s property, it should be used for work only. Moreover, some companies may install software that monitors your work.
+
+### Guided Practice
+In groups of 2 or 3, set up a remote pairing environment using only Slack (or another online tool) to communicate to simulate a remote collaboration. For this, an easy to set up free online pairing system is [CodeSandbox](https://codesandbox.io/) which only needs GitHub authentication to pair. See the instructions for pairing [here](https://www.ma-no.org/en/software/development/10-collaborative-coding-tools-for-remote-pair-programming), along with other choices. While still communicating online, pick a small project, hack away and see if you can complete it in the allotted time.
```

Comment:
Can you specifically mention to pair on one of the "daily code challenges"

Reply:
Yep